tests/cmaes_test.cpp

```diff
@@ -50,7 +50,7 @@ TEST_CASE("CMAESLogisticRegressionTest", "[CMAESTest]")
         responses, testResponses, shuffledResponses);
```

Comment:
Should we add a test that makes sure CMAES keeps the answers within the given bounds?

Reply:
Hi, @favre49 sorry for being late on this. Can you please elaborate on the test for this should be written? Should I consider an armadillo matrix which is out of bounds and then apply the transformation to see if the values are within the specified range? Sorry, that I didn't get the gist of what you were saying.

Reply:
I meant we should add a test that ensures the optimized coordinates are within the given bounds [-20,20]. Something like `REQUIRE(coordinates[i] <=20 && coordinates[i] >= -20)`

Reply:
Hi, @favre49 the test has been added kindly have a look whenever you get a chance. Thanks.